src/psxhle-emu-ifc.h

```diff
@@ -69,33 +69,33 @@
 #undef SysErrorPrintf
 
 #if HLE_DUCKSTATION_IFC
-#define SysErrorPrintf(fmt, ...)    Log_ErrorPrintf(fmt, ##__VA_ARGS__)
-#define SysPrintf(fmt, ...)         Log_VerbosePrintf(fmt, ##__VA_ARGS__)
-#define PSXBIOS_LOG(fmt, ...)       Log_VerbosePrintf(fmt, ##__VA_ARGS__)
-#define PSXBIOS_LOG_SPAM(fmt, ...)  Log_DebugPrintf(fmt, ##__VA_ARGS__)
-#define PSXBIOS_LOG_IRQ(fmt, ...)  Log_DebugPrintf(fmt, ##__VA_ARGS__)
+#define SysErrorPrintf(...)    Log_ErrorPrintf(__VA_ARGS__)
+#define SysPrintf(...)         Log_VerbosePrintf(__VA_ARGS__)
+#define PSXBIOS_LOG(...)       Log_VerbosePrintf(__VA_ARGS__)
+#define PSXBIOS_LOG_SPAM(...)  Log_DebugPrintf(__VA_ARGS__)
+#define PSXBIOS_LOG_IRQ(...)   Log_DebugPrintf(__VA_ARGS__)
 #endif
 
 #if !defined(PSXBIOS_LOG)
-#   define PSXBIOS_LOG(fmt, ...) (printf("[HLEBIOS] " fmt "\n", ##__VA_ARGS__), fflush(nullptr))
+#   define PSXBIOS_LOG(...) (printf(__VA_ARGS__), fflush(nullptr))
```

Comment:
I want to keep the `\n` otherwise log will be unreadable for me. And the tag is also important `[HLEBIOS]`. Logs are easily 1 millions of line, it is important to be easily grep-able

Reply:
Hum, this is strange that you have compilation issue here btw.
Log shall be defined in the previous DUCKSTATION relation block.